packages/uswds-core/src/styles/mixins/typography/typeset.scss

```diff
@@ -39,12 +39,8 @@ Sets:
   @include u-margin-y(0);
   clear: both;
 
-  * + & {
-    margin-top: 1.5em; // TODO: add as var to settings?
-  }
-
   + * {
-    margin-top: 1em;
```

Comment:
Was this not compiling properly?

Reply:
The `* + & {}` selector was improperly targeting classes when used as a mixin so we moved the specification directly to the selectors that were calling this mixin

The `+ * {}` selector is working as expected, we just swapped it for the variable